tests/spec/core/linter-rules/check-charset-spec.js

```diff
@@ -0,0 +1,72 @@
+describe("Core Linter Rule - 'check-charset'", () => {
+  const ruleName = "check-charset";
+  const config = {
+    lint: { [ruleName]: true },
+  };
+  let rule;
+  beforeAll(async () => {
+    rule = await new Promise(resolve => {
+      require([`core/linter-rules/${ruleName}`], ({ rule }) => resolve(rule));
+    });
+  });
+
+  it("checks if meta[charset] is set to utf-8", async () => {
+    const doc = document.implementation.createHTMLDocument("test doc");
+    doc.head.innerHTML = `
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width">
+      <title>ReSpec</title>
+    `;
+
+    const results = await rule.lint(config, doc);
+    expect(results.length).toEqual(0);
```

Comment:
Just a small nit... all the `toEqual` should be `toBe`. They are technically the same, `toBe` is semantically correct.